src_py/cursors.py

```diff
@@ -49,6 +51,18 @@
 
 class Cursor(object):
     def __init__(self, *args):
+        """Cursor() -> Cursor
+        Cursor(constant) -> Cursor
+        Cursor(Cursor) -> copies the Cursor object passed as an argument
+        Cursor(hotspot, Surface) -> Cursor
+        Cursor(size, hotspot, xormasks, andmasks) -> Cursor
```

Comment:
This should be in the same order, and the exact same as it is in the documentation

Reply:
This just got corrected